README.md

```diff
@@ -55,9 +55,14 @@ curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh
 # install poetry if not installed (ensure you've restarted your terminal after installation)
 pipx install poetry
 
+# disable any conda envs that might interfere with poetry's venv
+conda deactivate
```

Comment:
this should be used outside of venv and conda?

Reply:
Yeah they're independent w.r.t virtual envs, I've seen some people use conda as a substitute for `pyenv` which could cause issues but I think leaving a comment specific to this use case might be unneeded clutter